_src/_includes/sidenote.html

```diff
@@ -0,0 +1 @@
+<label for="{{ include.reference }}" class="margin-toggle sidenote-number"></label><input type="checkbox" id="{{ include.reference }}" class="margin-toggle"/><span class="sidenote">{{ include.content }}</span>
```

Comment:
Also, since all the sidenote references will be something like `sidenote-...`, let's make the `for` and `id` values be `sidenote-{{ ... }}` so that you don't have to repeat yourself all the time.